### PR TITLE
[MOSIP-32064, MOSIP-32456]

### PR DESCRIPTION
**Pull request by JanardhanBS-SyncByte**

### CODE REVIEW

File: mock-mv/src/main/java/io/mosip/mock/mv/queue/Listener.java [hunk 1]
```diff
@@ -138,51 +139,55 @@ public boolean consumeLogic(jakarta.jms.Message message, String mvAddress) {
...
 				}
-				Map<String,String> analytics=new HashMap<>();
-				analytics.put("primaryOperatorID", "110006");//logic needs to be implemented
+				Map<String, String> analytics = new HashMap<>();
+				analytics.put("primaryOperatorID", "110006");// logic needs to be implemented
 				analytics.put("primaryOperatorComments", "abcd");
-				candidateList.setCount(candidateList.getCandidates()!=null?candidateList.getCandidates().size():0);// logic needs to be implemented.
+				candidateList
+						.setCount(candidateList.getCandidates() != null ? candidateList.getCandidates().size() : 0);// logic
+																													// needs
```
Comment (LoganathanSekar7627) [inline mock-mv/src/main/java/io/mosip/mock/mv/queue/Listener.java:174]: Can you please correct the comment.

File: MockMDS/pom.xml [hunk 2]
```diff
@@ -365,14 +365,14 @@
 				<configuration>
 					<generateGitPropertiesFile>true</generateGitPropertiesFile>
 					<generateGitPropertiesFilename>
-						${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
+						${project.build.outputDirectory}/service-git.properties</generateGitPropertiesFilename>
```
Comment (LoganathanSekar7627) [inline MockMDS/pom.xml:368]: Should only change to a Service but not a library.

File: mock-sdk/pom.xml [hunk 3]
```diff
@@ -330,17 +329,15 @@
 				<configuration>
 					<generateGitPropertiesFile>true</generateGitPropertiesFile>
 					<generateGitPropertiesFilename>
-						${project.build.outputDirectory}/git.properties</generateGitPropertiesFilename>
+						${project.build.outputDirectory}/service-git.properties</generateGitPropertiesFilename>
```
Comment (LoganathanSekar7627) [inline mock-sdk/pom.xml:332]: Should be changed only for a service.

File: MockMDS/src/main/java/org/biometric/provider/CryptoUtility.java [hunk 4]
```diff
@@ -36,7 +36,8 @@ public class CryptoUtility {
 
 	private static BouncyCastleProvider provider;
 	private static final String ASYMMETRIC_ALGORITHM = "RSA/ECB/OAEPWITHSHA-256ANDMGF1PADDING";
-	private static final String SYMMETRIC_ALGORITHM = "AES/GCM/PKCS5Padding";
+	//private static final String SYMMETRIC_ALGORITHM = "AES/GCM/PKCS5Padding";
```
Comment (LoganathanSekar7627) [inline MockMDS/src/main/java/org/biometric/provider/CryptoUtility.java:39]: Can we remove the commented code.

**Review by LoganathanSekar7627 — CHANGES_REQUESTED** [review]

**Review by Prafulrakhade — APPROVED** [review]

**Pull request merged**